vit-servicing-station-tests/src/common/data/generator.rs

```diff
@@ -47,12 +51,13 @@ impl Snapshot {
         self.funds.iter().find(|x| x.id == id)
     }
 
-    pub fn token(&self) -> (APITokenData, String) {
-        self.token.clone()
+    pub fn any_token(&self) -> (String, APITokenData) {
+        let (hash, token) = self.tokens.iter().next().clone().unwrap();
```

Comment:
I'm not sure if this is gonna work as expected. HashMaps are not ordered by default by I think the iteration will always run in the same order for any of them (we just don't know which order). So, if we want `any_token` to always return the same token for any `snapshot` then it is ok. If we want to randomly return a token for the ones available we would need to use some other implementation.

Reply:
My assumption was that it does not matter at this point since, we have no validation of token atm. in future i will intorduce any_valid_token method  or similar